src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/Serialization/NamingStylePreferences.cs

```diff
@@ -139,252 +140,258 @@ public override int GetHashCode()
-    <SymbolSpecification ID=""2c07f5bf-bc81-4c2b-82b4-ae9b3ffd0ba4"" Name=""{CompilerExtensionsResources.Types}"">
-      <ApplicableSymbolKindList>
-        <TypeKind>Class</TypeKind>
-        <TypeKind>Struct</TypeKind>
-        <TypeKind>Interface</TypeKind>
-        <TypeKind>Enum</TypeKind>
-      </ApplicableSymbolKindList>
-      <ApplicableAccessibilityList>
-        <AccessibilityKind>Public</AccessibilityKind>
-        <AccessibilityKind>Internal</AccessibilityKind>
-        <AccessibilityKind>Private</AccessibilityKind>
-        <AccessibilityKind>Protected</AccessibilityKind>
-        <AccessibilityKind>ProtectedOrInternal</AccessibilityKind>
-        <AccessibilityKind>ProtectedAndInternal</AccessibilityKind>
-      </ApplicableAccessibilityList>
-      <RequiredModifierList />
-    </SymbolSpecification>
-    <SymbolSpecification ID=""5f3ddba1-279f-486c-801e-5c097c36dd85"" Name=""{CompilerExtensionsResources.Non_Field_Members}"">
-      <ApplicableSymbolKindList>
-        <SymbolKind>Property</SymbolKind>
-        <SymbolKind>Event</SymbolKind>
-        <MethodKind>Ordinary</MethodKind>
-      </ApplicableSymbolKindList>
-      <ApplicableAccessibilityList>
-        <AccessibilityKind>Public</AccessibilityKind>
-        <AccessibilityKind>Internal</AccessibilityKind>
-        <AccessibilityKind>Private</AccessibilityKind>
-        <AccessibilityKind>Protected</AccessibilityKind>
-        <AccessibilityKind>ProtectedOrInternal</AccessibilityKind>
-        <AccessibilityKind>ProtectedAndInternal</AccessibilityKind>
-      </ApplicableAccessibilityList>
-      <RequiredModifierList />
-    </SymbolSpecification>
-  </SymbolSpecifications>
-  <NamingStyles>
-    <NamingStyle ID=""87e7c501-9948-4b53-b1eb-a6cbe918feee"" Name=""{CompilerExtensionsResources.Pascal_Case}"" Prefix="""" Suffix="""" WordSeparator="""" CapitalizationScheme=""PascalCase"" />
-    <NamingStyle ID=""1ecc5eb6-b5fc-49a5-a9f1-a980f3e48c92"" Name=""{CompilerExtensionsResources.Begins_with_I}"" Prefix=""I"" Suffix="""" WordSeparator="""" CapitalizationScheme=""PascalCase"" />
-  </NamingStyles>
-  <NamingRules>
-    <SerializableNamingRule SymbolSpecificationID=""23d856b4-5089-4405-83ce-749aada99153"" NamingStyleID=""1ecc5eb6-b5fc-49a5-a9f1-a980f3e48c92"" EnforcementLevel=""Info"" />
-    <SerializableNamingRule SymbolSpecificationID=""2c07f5bf-bc81-4c2b-82b4-ae9b3ffd0ba4"" NamingStyleID=""87e7c501-9948-4b53-b1eb-a6cbe918feee"" EnforcementLevel=""Info"" />
-    <SerializableNamingRule SymbolSpecificationID=""5f3ddba1-279f-486c-801e-5c097c36dd85"" NamingStyleID=""87e7c501-9948-4b53-b1eb-a6cbe918feee"" EnforcementLevel=""Info"" />
-  </NamingRules>
-</NamingPreferencesInfo>
-";
+        private static readonly Lazy<string> s_defaultNamingPreferencesString = new(() => ConstructDefaultNamingPreferencesString());
+
+        private static string ConstructDefaultNamingPreferencesString()
+        {
+            using var _ = PooledStringBuilder.GetInstance(out var sb);
```

Comment:
Why not a simply a string as it was before?

Reply:
found it easier to work with personally. open to putting it back

Reply:
with raw string literals I would probably but it back